configs/samples/pjsip.conf.sample

```diff
@@ -658,8 +658,9 @@
                         ; identified.
                         ; "username": Identify by the From or To username and domain
                         ; "auth_username": Identify by the Authorization username and realm
-                        ; "ip": Identify by the source IP address
+                        ; "ip": Identify by the source (remote) IP address
```

Comment:
The sample configuration file needs an entire section talking about this new functionality, how to use it, how it works.

Reply:
Added an paragraph and an example. And there is the testcase as well which is in itself an example.
This is hard to describe in words as this is kind of an extreme use case.